### PR TITLE
#821 Background subtractor files added

### LINKED ISSUE #821: Remove Background from an Image using Python
# Description

Remove the background from an image in python using image manipulation libraries such as scikit-image, opencv etc.

## Language

- [X] Python
## Checklist:

- **Details on the Script**

| name            | about                                                  | labels  | assignees                                                      |
| --------------- | ------------------------------------------------------ | ------- | -------------------------------------------------------------- |
| Background Remover | Remove the background from an image using Python Image Processing | Python, Medium, Up For Grabs |  |

**Is your feature request related to a problem? Please describe.**: 
Many times we want the background to be removed from the image and the available background remover websites aren't generally free. Isn't it cool to make our own background remover script that can extract only the subject from the image?

This can be implemented by using Scikit image or Open CV like image processing libraries.

[This](https://chris-s-park.medium.com/image-background-removal-using-opencv-part-1-da3695ac66b6) and [this](https://flothesof.github.io/removing-background-scikit-image.html) briefs about the implementation but not limited to. 

If working on this, make sure the PR contains well-documented `README.md` explaining the working of the script in detail along with examples for the users to understand. Also follow the Coding Style of the project without fail.

Let me know if any assistance is required for the same.

**Describe the solution you'd like**: A clear and concise description of what you want to happen.

**Additional context**: Add any other context or screenshots about the feature request here.

### PR DESCRIPTION
**Pull request by iamakkkhil**
# Description

This python script lets you remove background form image by keeping only foreground in focus. By using Grabcut Algorithm it makes easy to do the task.

Fixes #821

Replace `issue_no` with the issue number which is fixed in this PR

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works

### CODE REVIEW

File: Python/Background_Subtractor/README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+
+## Output:
+![Image](https://i.imgur.com/SCf4XRj.jpg)
+
+## Author
+[🛡 Akhil Bhalerao 🛡 ](https://github.com/iamakkkhil)
+
+## Credits
+https://www.geeksforgeeks.org/python-foreground-extraction-in-an-image-using-grabcut-algorithm/
```
Comment (HarshCasper) [inline Python/Background_Subtractor/README.md:48]: How come that you are the author of the script and yet you share credits with an external resource?

**Review by HarshCasper — CHANGES_REQUESTED** [review]
The Script looks quite good to me. One question and I guess it would be ready to merge. 

@vybhav72954 can have a look as well.

File: Python/Background_Subtractor/README.md (hunk 1 above)
Comment (iamakkkhil) [reply Python/Background_Subtractor/README.md:48]: I didn't knew about grabcut algorithm, so I have read  and understood theory from gfg and then implemented it. 
What should I do now ? Should I remove the credits link or give it a new heading? @HarshCasper @vybhav72954 @sohamsshah

**Comment by iamakkkhil** [comment]
Can you please add the level tag as well .

File: Python/Background_Subtractor/README.md (hunk 1 above)
Comment (HarshCasper) [reply Python/Background_Subtractor/README.md:48]: I guess let the credits be there. Just add it as a hyperlink over there. Appreciate your honesty over here :)

**Comment by HarshCasper** [comment]
> Can you please add the level tag as well .

Let @vybhav72954 do a review once. We will merge it and add the level tag

File: Python/Background_Subtractor/README.md (hunk 1 above)
Comment (iamakkkhil) [reply Python/Background_Subtractor/README.md:48]: 😊

File: Python/Background_Subtractor/BG_Subtractor.py [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+    :param path: image location path
+    :return:
+    """
+    img = cv2.imread(path)
+    dim = (512, 512)
+    resized_image = cv2.resize(img, dim, interpolation=cv2.INTER_AREA)
+    return resized_image
+
+
+# mouse callback function
```
Comment (vybhav72954) [inline Python/Background_Subtractor/BG_Subtractor.py:23]: Let's remove these comments shall we, you have written an awesome docstring here.

**Review by vybhav72954 — CHANGES_REQUESTED** [review]
One teeny tiny change, because am a nitpicker sometimes.

And hats off to the honesty here, really love when people give credit to their sources.

**Comment by vybhav72954** [comment]
@HarshCasper LGTM

**Review by vybhav72954 — APPROVED** [review]

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**